cli/ballerina-cli/src/main/resources/cli-help/ballerina-run.help

```diff
@@ -1,94 +1,78 @@
+       ballerina-run - Compile and run the current package
 
 SYNOPSIS
-       bal run 
-       bal run <executable-jar>
-       bal run [--target-dir] <target-directory-path>
-       bal run [--offline] [--experimental] [--debug] [--observability-included] 
-               <ballerina-file-path> 
-       bal run [--offline] [--experimental] [--debug] [--observability-included] 
-               <ballerina-file-path> [--] <args...>
-       bal run [--offline] [--experimental] [--debug] [--observability-included] 
-               <ballerina-package-path> [--] <args...>
+       bal run [OPTIONS] [<package>|<source-file>|<jar-file>] [--] <args...>
+
 
 DESCRIPTION
-       Build and execute the given standalone Ballerina file, package, or program.
+       Compile the current package and run it. This command does not produce
+       an executable program.
 
-       If a standalone Ballerina file or a package directory is given, this
-       command builds and runs it. Compilation is done internally and the
-       executable will not be created.
+       The run command executes the program when a jar file produced by 'bal build'
+       is given as the argument. If not, compiles and runs the given package
+       or single bal file.
 
-       You may use the 'bal build' command to compile a source and
-       provide the generated binary (JAR) file to the run command.
-       The binary runs much faster than a source file because running a source
-       file requires a compilation phase.
+       Ballerina program consists of one or more modules; one of these modules
+       is distinguished as the root module, which is the default module of
+       current package.
 
-       You must have either a public 'main' function or a service(s) or both
-       in order to run a program or a package.
+       Ballerina program execution consists of two consecutive phases.
+       The initialization phase initializes all modules of a program one after another.
+       If a module defines a function named 'init()', It'll be invoked during this phase.
+       If the root module of the program defines a public function named 'main()',
+       then it'll be invoked.
 
-       When both a 'main' function and a service(s) are present, this command executes
-       the 'main' function first and then starts the services.
+       If the initialization phase of program execution completes successfully,
+       then execution proceeds to the listening phase. If there are no module listeners,
+       then the listening phase immediately terminates successfully. Otherwise,
+       the listening phase initializes module listeners.
 
-       Note: Running `.bal` files, which belong to a Ballerina package is not allowed.
+       A service declaration is syntactic sugar for creating a service object and
+       attaching it to the module listener specified in the service declaration.
 
+       Note: Building individual '.bal' files of the current package is not allowed.
```

Comment:
Shouldn't this be
```suggestion
       Note: Running individual '.bal' files of the current package is not allowed.
```

Same comment as previously re: adding "current" here also.